addons/web_editor/static/src/scss/web_editor.backend.scss

```diff
@@ -38,8 +38,8 @@
     table.table.table-bordered {
         table-layout: fixed;
     }
-    a {
-        // Ensure the link is visible enough in the editor.
+    a:not(.btn) {
+        // Ensure non-button links are visible enough in the editor.
```

Comment:
About the commit message, you should probably read this: https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html

Also, I think this particular commit should mention the original one.

Considering all the above, I suggest this message:
```js
[FIX] web_editor: button links should not use oe-link-color

Commit[1] intended to change the color of links in the editor to make them more
visible by making them a slightly lighter color. However, it did so in a way
that also applied to button links, which already had different colors. In the
case of primary buttons, it actually made the links less readable in the end.

This commit excludes buttons from the original css rule so that they keep their
special style while still maintaining the intent of the original commit.

[1]: 47c73c7213c06a5bbb496031707e1ec20e060bf1
```

Reply:
What a nice message @dmo-odoo . Should I mention you as the author since I'm definitely copy pasting it ?

Reply:
Don't worry. Authorship of code matters but authorship of commit messages, not so much. ;-)

Reply:
@dmo-odoo Done.